mne/commands/mne_browse_raw.py

```diff
@@ -89,6 +90,12 @@ def run():
     lowpass = options.lowpass
     filtorder = options.filtorder
     clipping = options.clipping
+    if isinstance(clipping, str):
+        clipping = None if clipping.lower() == 'none' else clipping
+    try:
+        clipping = float(clipping)  # allow float and convert it
+    except ValueError:
+        pass
```

Comment:
`float(x)` will raise `TypeError` if `x is None`. We should explicitly handle this case as we're setting `clipping = None` in certain situations.


```suggestion


    if isinstance(clipping, str):
        clipping = None if clipping.lower() == 'none' else clipping

    if clipping is not None:
        try:
            clipping = float(clipping)  # allow float and convert it
        except ValueError:
            pass
```

Reply:
done

(Didn't use the suggestion because it had PEP8 errors)